lib/timer.py

```diff
@@ -117,5 +111,17 @@ def run_jobs(cfg: UAConfig, current_time: datetime):
 if __name__ == "__main__":
     cfg = UAConfig()
     current_time = datetime.utcnow()
-    setup_logging(logging.INFO, logging.DEBUG)
+
+    # The ua-timer logger should log everything to its file
+    setup_logging(
+        logging.CRITICAL,
+        logging.DEBUG,
+        log_file=cfg.timer_log_file,
+        logger=LOG,
+    )
+    # Make sure the ua-timer logger does not generate double logging
+    LOG.propagate = False
+    # The root logger should log any error to the timer log file
+    setup_logging(logging.CRITICAL, logging.ERROR, log_file=cfg.timer_log_file)
```

Comment:
Yes, we do have the exception, but it would ignore all errors in the process, handled by the root logger. Besides that, this setup_logging call is ensuring we log the "Disabling" message (which is an INFO) when not in GCE.